src/ripple/ledger/impl/View.cpp

```diff
@@ -24,6 +24,7 @@
 #include <ripple/basics/contract.h>
 #include <ripple/basics/Log.h>
 #include <ripple/basics/StringUtilities.h>
+#include <ripple/protocol/Feature.h>
```

Comment:
This `#include` looks like it is not necessary.  Both my clang non-unity and unity builds succeed without it.